public/data/umaresso.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "Umar Khatab",
+  "type": "personal",
+  "bio": "React JS Developer",
+  "avatar": "https://github.com/umaresso.png",
+  "links": [
+    {
+      "name": "Connect on GitHub",
+      "url": "https://github.com/umaresso",
+      "icon": "github"
+    },
+    {
+      "name": "Let's Connect on Twitter",
+      "url": "https://twitter.com/umarkhatab465",
+      "icon": "twitter"
+    },
+    {
+      "name": "Learn about Front End Developement",
+      "url": "https://www.youtube.com/channel/UC61r1YBoNKSiI-c5ezj_FJQ",
+      "icon": "youtube"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://linkedin.com/in/umarkhatab465",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started YouTube",
+      "image": "https://www.youtube.com/channel/UC61r1YBoNKSiI-c5ezj_FJQ",
+      "date": "Feb 2022",
```

```suggestion
      "date": "February 2022",
```

Instead of the short form, its better to put the full month in 😆